### PR TITLE
Corrects the description of the reuse time

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

The API document says that the reuse time is hard coded, which it is not. It also gives  an incorrect description of what parameters that are taken into account when comparing two requests.

## Changes

Description is corrected.

## How to test this PR

This PR only updates a document, not the implementation.

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -658,8 +658,15 @@ An object with the following properties:
 
 A *test id*. 
 
-If the test has been run with the same domain name within an interval of 10 mins (hard coded), 
-then the new request does not trigger a new test, but returns with the results of the last test
+If a test has been run with the same parameters (as listed below) not more than
```
Comment (mattias-p) [inline docs/API.md:661]: There's an ambiguity here between when a test is requested and when it is started. I think "requested" would be a better word than "run".

**Review by mattias-p — APPROVED** [review]

File: docs/API.md (hunk 1 above)
Comment (matsduf) [reply docs/API.md]: I have updated. @mattias-p, please re-review.

**Review by mattias-p — APPROVED** [review]

**Comment by matsduf** [comment]
@blacksponge, can you review this?

**Review by blacksponge — APPROVED** [review]

**Comment by matsduf** [comment]
@blacksponge, I just resolved the conflict. Can you approve again?

**Review by blacksponge — APPROVED** [review]

**Pull request merged**